The job **`mingw-check-tidy`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/13847287217/job/38748258937) [(plain)](https://github.com/rust-lang/rust/commit/82a79b99e1279ad72cb93c2ad10c6be1facaca4a/checks/38748258937/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
info: removing rustup binaries
info: rustup is uninstalled
##[group]Image checksum input
mingw-check-tidy
# We use the ghcr base image because ghcr doesn't have a rate limit
# and the mingw-check-tidy job doesn't cache docker images in CI.
FROM ghcr.io/rust-lang/ubuntu:22.04

ARG DEBIAN_FRONTEND=noninteractive
RUN apt-get update && apt-get install -y --no-install-recommends \
  g++ \
  make \
---

COPY host-x86_64/mingw-check/validate-toolstate.sh /scripts/
COPY host-x86_64/mingw-check/validate-error-codes.sh /scripts/

# NOTE: intentionally uses python2 for x.py so we can test it still works.
# validate-toolstate only runs in our CI, so it's ok for it to only support python3.
ENV SCRIPT TIDY_PRINT_DIFF=1 python2.7 ../x.py test \
           --stage 0 src/tools/tidy tidyselftest --extra-checks=py,cpp
#
# This file is autogenerated by pip-compile with Python 3.10
# by the following command:
#
#    pip-compile --allow-unsafe --generate-hashes reuse-requirements.in
---
#12 2.905 Building wheels for collected packages: reuse
#12 2.906   Building wheel for reuse (pyproject.toml): started
#12 3.121   Building wheel for reuse (pyproject.toml): finished with status 'done'
#12 3.121   Created wheel for reuse: filename=reuse-4.0.3-cp310-cp310-manylinux_2_35_x86_64.whl size=132720 sha256=0c2fd2aaf7b0bf8d6e131220aff14712a774c2ca462f3204d25460cbcf610b63
#12 3.122   Stored in directory: /tmp/pip-ephem-wheel-cache-czt228mi/wheels/3d/8d/0a/e0fc6aba4494b28a967ab5eaf951c121d9c677958714e34532
#12 3.124 Successfully built reuse
#12 3.124 Installing collected packages: boolean-py, binaryornot, tomlkit, reuse, python-debian, markupsafe, license-expression, jinja2, chardet, attrs
#12 3.529 Successfully installed attrs-23.2.0 binaryornot-0.4.4 boolean-py-4.0 chardet-5.2.0 jinja2-3.1.4 license-expression-30.3.0 markupsafe-2.1.5 python-debian-0.1.49 reuse-4.0.3 tomlkit-0.13.0
#12 3.529 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
#12 4.061 Collecting virtualenv
#12 4.105   Downloading virtualenv-20.29.3-py3-none-any.whl (4.3 MB)
#12 4.164      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 4.3/4.3 MB 76.4 MB/s eta 0:00:00
#12 4.225 Collecting filelock<4,>=3.12.2
#12 4.229   Downloading filelock-3.17.0-py3-none-any.whl (16 kB)
#12 4.265 Collecting platformdirs<5,>=3.9.1
#12 4.270   Downloading platformdirs-4.3.6-py3-none-any.whl (18 kB)
#12 4.296 Collecting distlib<1,>=0.3.7
#12 4.300   Downloading distlib-0.3.9-py2.py3-none-any.whl (468 kB)
#12 4.307      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 469.0/469.0 KB 91.7 MB/s eta 0:00:00
#12 4.388 Installing collected packages: distlib, platformdirs, filelock, virtualenv
#12 4.575 Successfully installed distlib-0.3.9 filelock-3.17.0 platformdirs-4.3.6 virtualenv-20.29.3
#12 4.575 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
#12 DONE 4.7s

#13 [7/8] COPY host-x86_64/mingw-check/validate-toolstate.sh /scripts/
#13 DONE 0.0s
---
DirectMap4k:      126912 kB
DirectMap2M:     8261632 kB
DirectMap1G:    10485760 kB
##[endgroup]
Executing TIDY_PRINT_DIFF=1 python2.7 ../x.py test            --stage 0 src/tools/tidy tidyselftest --extra-checks=py,cpp
+ TIDY_PRINT_DIFF=1 python2.7 ../x.py test --stage 0 src/tools/tidy tidyselftest --extra-checks=py,cpp
##[group]Building bootstrap
    Finished `dev` profile [unoptimized] target(s) in 0.05s
##[endgroup]
WARN: currently no CI rustc builds have rustc debug assertions enabled. Please either set `rust.debug-assertions` to `false` if you want to use download CI rustc or set `rust.download-rustc` to `false`.
WARN: `rust.debug-assertions = true` will prevent downloading CI rustc as alt CI rustc is not currently built with debug assertions.
---
fmt check
fmt: checked 5905 files
tidy check
tidy: Skipping binary file check, read-only filesystem
##[error]tidy error: /checkout/tests/codegen/issues/issue-129795.rs:5: line longer than 100 chars
removing old virtual environment
creating virtual environment at '/checkout/obj/build/venv' using 'python3.10' and 'venv'
creating virtual environment at '/checkout/obj/build/venv' using 'python3.10' and 'virtualenv'
Requirement already satisfied: pip in ./build/venv/lib/python3.10/site-packages (25.0.1)
linting python files
All checks passed!
checking python file formatting
26 files already formatted
checking C++ file formatting
some tidy checks failed
Command has failed. Rerun with -v to see more details.
Build completed unsuccessfully in 0:01:50
  local time: Fri Mar 14 00:40:39 UTC 2025
  network time: Fri, 14 Mar 2025 00:40:39 GMT
##[error]Process completed with exit code 1.
Post job cleanup.
```

</details>

--- a/tests/codegen/issues/issue-129795.rs
+++ b/tests/codegen/issues/issue-129795.rs
@@ -2,7 +2,8 @@
 //@ min-llvm-version: 20
 #![crate_type = "lib"]
 
-// Ensure that a modulo operation with an operand that is known to be a power-of-two is properly optimized.
+// Ensure that a modulo operation with an operand that is known to be
+// a power-of-two is properly optimized.
 
 // CHECK-LABEL: @modulo_with_power_of_two_divisor
 // CHECK: add i64 %divisor, -1

